introduction.md

```diff
@@ -1 +1,8 @@
 Hi, I'm doing a code school tutorial for Github - which is my favourite path so far, because I use git all the time, but there are millions of things I didn't know you could do.  But that doesn't count, apparently, so Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?